13-foundations-bootstrapping.Rmd

```diff
@@ -310,11 +311,11 @@ ggplot(bsprop_tap, aes(x = bsprop)) +
 (ref:tappers-bs-sim-cap) The original listener-tapper data is bootstrapped 10,000 times. Each simulation creates a sample where the probability of being correct is $\hat{p} = 3/120.$ The 2.5 percentile proportion is 0 and the 97.5 percentile is 0.0583. The result is that we are confident that, in the population, the true percent of people who can guess correctly is between 0% and 5.83%.
 
 ::: {.guidedpractice data-latex=""}
-Do the data provide "statistically significant" evidence against the claim that 50% of listeners can guess the tapper's tune?[^foundations-bootstrapping-3]
+Do the data provide strong evidence against the claim that 50% of listeners can guess the tapper's tune?[^foundations-bootstrapping-3]
 :::
 
-[^foundations-bootstrapping-3]: Because 50% is not in the interval estimate for the true parameter, there is statistically significant evidence.
-    Indeed, the data provide strong evidence that the chance a listener will guess the correct tune is less than 50%.
+[^foundations-bootstrapping-3]: Because 50% is not in the interval estimate for the true parameter, we can say that there is strong evidence against the hypothesis that 50% of listeners can guess the tune.
```

Comment:
```suggestion
[^foundations-bootstrapping-3]: Because 50% is not in the interval estimate for the true parameter, we can say that there is convincing evidence against the hypothesis that 50% of listeners can guess the tune.
```